src/modules/Search/Search.js

```diff
@@ -176,6 +181,9 @@ export default class Search extends Component {
     minCharacters: 1,
     noResultsMessage: 'No results found.',
     showNoResults: true,
+    input: {
```

Comment:
Once the shorthand changes are made, the default value here would just be `text`.  That will ensure the `Input.create()` shorthand creates a `type='text'` input by default.

We don't explicitly need to set the `type`, it is just that we need a non-nil value to work with.  Otherwise, the `Input.create()` shorthand factory would not create an element.  All factories return null if passed null/undefined.